src/desdemona/functions/sample_functions.clj

```diff
@@ -1,21 +1,23 @@
 (ns desdemona.functions.sample-functions
-  (:require [clojure
-             [walk :refer [postwalk]]]))
+  (:require
+   [clojure.core.match :refer [match]]))
 
 ;;; Defines functions to be used by the peers. These are located
 ;;; with fully qualified namespaced keywords, such as
-;;; desdemona.functions.sample-functions/format-line
+;;; desdemona.functions.sample-functions/add-message-origin
```

Comment:
I agree about the comment. About the namespace, do you mean the "sample-functions" name? Because yeah at some point we shouldn't call everything in here "sample" any more.